06_modules_lab/01.py

```diff
@@ -1,4 +1,9 @@
-""" Write a program that takes a count
-from sys.argv import and prints "Hello Python"
-count times
-"""
+import sys
+

```

Comment:
Checking the inverse condition yields nicer code. Consider:
```
if ...:
  print "error"
  sys.exit(1)

# code continues here unindented
```

The advantage is you get some short lines of indented code to handle the error, and most of your program is written nicely closed to the left